pkg/nb/rpc_ws.go

```diff
@@ -321,7 +320,7 @@ func (c *RPCConnWS) ReadMessage() (*RPCMessage, error) {
 
 	// Read message buffers if any
 	if msg.Buffers != nil && len(msg.Buffers) > 0 {
-		buffers, err := ioutil.ReadAll(reader)
+		buffers, err := io.ReadAll(reader)
```

Comment:
Not sure but I think we are missing to close the fd here as well 🤔

Reply:
Looks like we're OK here. The WS  RPC logic [IMHO could be crispier 😉 ](https://github.com/noobaa/noobaa-operator/pull/811) however, we don't necessarily want to close the WebSocket connection after reading each message, since the client might want to send us another one. The WS.connection is closed [here](https://github.com/noobaa/noobaa-operator/blob/60628671e4f05f55b49af1ab6d58ff689d1536d0/pkg/nb/rpc_ws.go#L162-L167). WDYT @utkarsh-pro ?

Reply:
Oh right, that makes sense. Thank you! 😄